stdlib/pyexpat/__init__.pyi

```diff
@@ -27,7 +28,10 @@ class XMLParserType(object):
     def SetBase(self, __base: Text) -> None: ...
     def GetBase(self) -> Optional[str]: ...
     def GetInputContext(self) -> Optional[bytes]: ...
-    def ExternalEntityParserCreate(self, __context: Optional[Text], __encoding: Text = ...) -> XMLParserType: ...
+    if sys.version_info >= (3, 8):
```

Comment:
I'm running 3.7 and the argument is already positional-only. Looking at the C code I suspect it's always been this way: the pre-Argument Clinic code in https://github.com/python/cpython/commit/d0aeda8f0ffbe4a7de9b492de9713d672dc8592d#diff-0534aca0d881f2dd1100e875602f98da60bf20ce127564896dc1e8a92a6fad2cL997 already didn't support kwargs.